docs/product-and-design/copy-delivery.md

```diff
@@ -10,34 +10,49 @@ Translation strings include all application copy, including:
 
-## Cal-ITP Benefits Application Copy
+Developers create the 'django.po' message files from copy created and implemented by Design and Product. Design and Product use the tools Figma, Ditto, and Crowdin to craft product copy, manage all copy and variantions of copy (e.g. agency-specific copy or Spanish translation copy), and translate copy.
 
-The human-readable version of the English and Spanish translation strings for the application are delivered to Design and Engineering by Product, and live at this link: [Cal-ITP Benefits Application Copy](https://docs.google.com/spreadsheets/d/1_Gi_YbJr4ZuXCOsnOWaewvHqUO1nC1nKqiVDHvw0118/edit#gid=0).
+## Tools used in Cal-ITP Benefits Application Copy
 
-By tabs:
+### Figma
 
-- `EN-USA` tab contains all copy for English, which each row representing a page. This copy uses a sample agency, called California State Transit (CST) with an Agency Card. This copy is used in Figma.
-- `forTranslation` and `All Agencies` tab contains the English and Spanish translation side by side, with agency-specific copy.
+Figma is our primary design tool and is the source of truth for all design decisions, concepts, and directions for the Benefits app and all Cal-ITP sites. We have Figma Organization which includes developer mode, version history, and branching–a tool to explore new design directions.
+
+The development team prefers not to have admin or editor capabilities so as not to accidentally interfere with any designs. Currently there is no formal process within Figma for handoff. Instead handoffs occur over the course of two meetings, a review of the designs and then what we call “issue-fest,” or a chance for developers to create Github issues while designers can weigh in and answer questions.
+
+The dev team and design largely collaborate on designs through Slack and comments directly in Figma. Design aims to address Figma comments as soon as possible or within one day.
+
+### Ditto
+
+The human-readable version of the English and Spanish translation strings for the application are delivered to Product and Engineering by Design, and are managed in the Ditto application. [Cal-ITP Benefits](https://app.dittowords.com/projects/65e119db3f806ee92198378a/page/5766:22626).
+
+Ditto is our copy management system for Benefits and the Cal-ITP sites. It syncs directly with Figma and uses many of the same conventions found in Figma, including components and variables (therefore, it is important to be specific when referring to a Ditto component or a Figma variable).
+
+The Benefits project within Ditto is fully synced with Figma. Product and design may use either Figma or Ditto as their source of truth on copy, though it is recommended that Developers use Figma for all final copy. Compiler is considering using the Github integration that comes with Ditto, but has not formally integrated Ditto into their workflow as of yet.
```

Comment:
Question for @srhhnry - It makes sense to me for developers to use Figma to copy and paste English copy, but for getting Spanish copy, we'd have to go into Ditto, right? Or is there a way to check Figma for Spanish copy?

Reply:
- For viewing copy in Spanish in Figma, one needs to have the Ditto plugin active. From there, select a frame and the Ditto plugin should automatically move to the Selected tab, and within that tab there's a dropdown to preview different variants. 

<img width="1271" alt="image" src="https://github.com/user-attachments/assets/616e52aa-a582-4fda-98c7-9c7d4ee1d77c">

Everyone has varying levels of permission, so if it is not possible to view that tab within the Ditto plugin, then yes I recommend using Ditto. 

(should the above^  go into the documentation as well?)

Reply:
This is _exactly_ the kind of instruction that should go in the Documentation. A good rule of thumb is: If you think you might have to copy/paste the same instructions to more than 1 person, it should be in the Docs. You could format is as 1, 2, 3 numbered instructions.